executor/prepared.go

```diff
@@ -295,6 +297,7 @@ func ResetStmtCtx(ctx sessionctx.Context, s ast.StmtNode) {
 			sc.Priority = mysql.LowPriority
 		}
 	case *ast.InsertStmt:
+		sc.IgnoreErr = stmt.IgnoreErr
```

Comment:
Do we need to add this code to `LoadDataStmt`?

Reply:
Does LoadData have IgnoreErr?

Reply:
I think this part is used for prepared statement. The LoadDataStmt is a little different from other statement. I add one line 551 write.go. So there is no need to add this here.

Reply:
Before every execution, we will do `ResetStmtCtx`.